libpod/container_internal.go

```diff
@@ -30,7 +30,9 @@ import (
 
 const (
 	// name of the directory holding the artifacts
-	artifactsDir = "artifacts"
+	artifactsDir       = "artifacts"
+	execDirPermission  = 0775
```

Comment:
This seems a little loose? Why 7 for group?

Reply:
I honestly don't remember why I set it this way. Got better suggestions? 0755?

Reply:
755 sounds a lot safer IMO